quixstreams/app.py

```diff
@@ -628,6 +628,17 @@ def get_consumer(self, auto_commit_enable: bool = True) -> Consumer:
             extra_config=self._config.consumer_extra_config,
         )
 
+    def get_store(
+        self, topic: str, store_name: str = DEFAULT_STATE_STORE_NAME
+    ) -> Store:
+        """
+        Get a store for given name and topic
+        :param topic: topic name
+        :param store_name: store name
+        :return: instance of `Store`
+        """
+        return self._state_manager.get_store(topic, store_name)
+
```

Comment:
This method doesn't seem to be used anywhere except in tests. 
If it's not needed, let's remove it from the App to avoid increasing the number of public methods.